src/preferences.html

```diff
@@ -5,6 +5,7 @@
     <title data-i18n="$Preferences.title">Time to Leave - Preferences</title>
     <link rel="stylesheet" href="../node_modules/bootstrap/dist/css/bootstrap.min.css">
     <link rel="stylesheet" href="../css/styles.css">
+    <link rel="stylesheet" href="../node_modules/@fortawesome/fontawesome-free/css/all.min.css">
```

Comment:
I think it's a good practice to leave the custom CSS as the last one to be sourced:

```suggestion
    <link rel="stylesheet" href="../node_modules/@fortawesome/fontawesome-free/css/all.min.css">
    <link rel="stylesheet" href="../css/styles.css">
```